lib/module.js

```diff
@@ -90,6 +91,16 @@ function httpModule(_moduleOptions) {
       typeof options.proxy === 'object' ? options.proxy : {}
     ])
   }
+  // Add `ky` to build.transpile
+  this.options.build = this.options.build || {}
+  this.options.build.transpile = this.options.build.transpile || {}
+  // transpile only for non-modern build
+  // istanbul ignore if
+  if (semver.gte(this.nuxt.constructor.version, 'v2.9.0')) {
+    this.options.build.transpile.push(({ isLegacy }) => isLegacy ? 'ky' : null)
```

Comment:
Can be simplified to `isLegacy && 'key'`

Reply:
Indeed, updated :)